backend/bang-ggood/src/test/java/com/bang_ggood/checklist/service/ChecklistServiceTest.java

```diff
@@ -378,6 +379,32 @@ void readCustomChecklistQuestions() {
         Assertions.assertThat(selectedCount).isEqualTo(questions.size());
     }
 
+    @DisplayName("좋아요된 체크리스트 리스트 조회 성공")
+    @Test
+    void readLikedUserChecklistsPreview() {
+        //given
+        checklistRepository.saveAll(
+                List.of(ChecklistFixture.CHECKLIST1_USER1,
+                        ChecklistFixture.CHECKLIST2_USER1,
+                        ChecklistFixture.CHECKLIST3_USER1)
+        );
+        checklistLikeRepository.saveAll(
+                List.of(ChecklistFixture.CHECKLIST_LIKE_1,
+                        ChecklistFixture.CHECKLIST_LIKE_2)
+        );
+
+        //when
+        List<UserChecklistPreviewResponse> checklists =
+                checklistService.readLikedUserChecklistsPreview(USER1).checklists();
+
+        //then
+        assertAll(
+                () -> assertThat(checklists.size()).isEqualTo(2),
+                () -> assertThat(checklists.get(0).checklistId()).isEqualTo(1),
+                () -> assertThat(checklists.get(1).checklistId()).isEqualTo(2)
```

Comment:
```suggestion
     () -> assertThat(checklists.get(0).checklistId()).isEqualTo(ChecklistFixture.CHECKLIST1_USER1.getId()),
     () -> assertThat(checklists.get(1).checklistId()).isEqualTo(ChecklistFixture.CHECKLIST2_USER1.getId())
```

Checklist에서 id를 꺼내와서 확인하는게 더 안전할 것 같아요~

Reply:
id가 같다면 equals()를 쓰는 게 더 좋을 것 같네요

Reply:
`readLikedChecklistsPreview`의 반환타입이 dto라 기존 Fixture랑 equals가 안돼서 id로 비교했습니다!